addons/mail/tests/test_ir_mail_server.py

```diff
@@ -441,3 +444,253 @@ def test_personal_mail_server(self):
             message = self._build_email(mail_from='test@custom_domain.com')
             IrMailServer.send_email(message, mail_server_id=self.mail_server_user.id)
         self.assertFalse(self.emails)
+
+    @mute_logger('odoo.models.unlink')
+    def test_personal_mail_server_limit(self):
```

Comment:
A bit unrelated but quand même: why not adding a new test class (like TestPersonalServer) and moving this test, `` test_personal_mail_server``, and maybe other tests added during the rush-merge in that class ? It helps having classes, as it is easier to focus on some specific features :)